docs/tables/aws_ssoadmin_account_assignment.md

```diff
@@ -0,0 +1,57 @@
+# Table: aws_ssoadmin_account_assignment
+
+Contains information about AWS SSO account assignments (an assignment of a SSO permission set to an Identity Store User/Group in an Account).
+
+## Examples
+
+### Assignments for a specific permission set and account
+
+```sql
+select
+    permission_set_arn,
+    target_account_id,
+    identity_store_id,
+    principal_type,
+    principal_id,
+from
+    aws_ssoadmin_account_assignment
+where
+    permission_set_arn = 'arn:aws:sso:::permissionSet/ssoins-0123456789abcdef/ps-0123456789abcdef'
+    and target_account_id = '012347678910'
```

Comment:
```suggestion
select
  permission_set_arn,
  target_account_id,
  identity_store_id,
  principal_type,
  principal_id,
from
  aws_ssoadmin_account_assignment 
where
  permission_set_arn = 'arn:aws:sso:::permissionSet/ssoins-0123456789abcdef/ps-0123456789abcdef' 
  and target_account_id = '012347678910';
```

@janslow we typically format our queries using https://www.freeformatter.com/sql-formatter.html#before-output to maintain consistency across all the table docs 👍.